lib/HTFeed/BackupExpiration.pm

```diff
@@ -55,10 +59,9 @@ SQL
         AND namespace=?
         AND id=?
         AND version < DATE_FORMAT(DATE_SUB(NOW(), INTERVAL 180 DAY),"%Y%m%d%H%i%S")
-        AND version!=?
+        ORDER BY version DESC LIMIT 1,999999999
```

Comment:
This works, but I agree we don't really want the magical number here... Could we instead just `ORDER BY version DESC` and skip the first row of the results on the application side?